examples/slider/slider-color-viewer.html

```diff
@@ -0,0 +1,422 @@
+                  </svg>
+                </div>
+                <button class="change inc" aria-label="increase blue one" tabindex="-1">
+                  <svg width="32" height="38">
+                    <circle class="hc-background" cx="14" cy="24" r="14"/>
+                    <circle class="background" cx="14" cy="24" r="12"/>
+                    <line x1="10" y1="24" x2="18" y2="24"/>
+                    <line x1="14" y1="20" x2="14" y2="28"/>
+                  </svg>
+                </button>
+                <button class="change inc10" aria-label="increase blue ten" tabindex="-1">
+                  <svg width="32" height="38">
+                    <circle class="hc-background" cx="14" cy="24" r="14"/>
+                    <circle class="background" cx="14" cy="24" r="12"/>
+                    <line x1="5" y1="24" x2="13" y2="24"/>
+                    <line x1="9" y1="20" x2="9" y2="28"/>
+                    <line x1="15" y1="24" x2="23" y2="24"/>
+                    <line x1="19" y1="20" x2="19" y2="28"/>
+                  </svg>
+                </button>
+              </div>
+
+              <h3>Color View Box</h3>
+              <div class="color-info">
+                <div class="color-box"></div>
+                <label>
+                  Color (HEX):
+                  <input type="text" readonly class="color-value-hex" value="">
+                </label>
+                <label>
+                  Color (RGB):
+                  <input type="text" readonly class="color-value-rgb" value="">
+                </label>
+              </div>
+            </div>
+          </div>
+          <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+    </section>
+
+  <section>
+      <h2>Accessibility Features</h2>
+      <ul>
+        <li>
+          The three sliders are wrapped in a group to help assistive technology users understand that all three elements together represent a color picker.
+        </li>
+        <li>Use of SVG enables high contrast support by including additional background <code>rects</code> and <code>circle</code> elements to provide contrast for the focus and hover border styling of the buttons and sliders.</li>
+        <li>The placement of the slider value above the thumb supports user's with low vision by allow them to more easily see the value of the slider as they move the thumb.</li>
+        <li>The use of the <code>currentColor</code> for the slider value supports the visibility of the value in operating system high contrast modes.</li>
+        <li>
+          The increase and decrease buttons are not contained within the <code>div</code> with role <code>slider</code> so they can be separately focusable by users of touch screen readers.
+          However, they are excluded from the page <kbd>Tab</kbd> sequence with <code>tabindex="-1"</code> because they are redundant with the arrow key support provided to keyboard users.
```

Comment:
```suggestion
          However, they are excluded from the page <kbd>Tab</kbd> sequence with <code>tabindex="-1"</code> because they are redundant with the arrow key support provided for keyboard users.
```